Document the deprecated component file changes for 21.0 (#952)
Dyalog/documentation: windows-installation-and-configuration-guide/docs/configuration-parameters/dyalog-extvar-supported.md

# DYALOG_EXTVAR_SUPPORTED

This parameter specifies whether support for external variables is enabled.

The values of the parameter are:

|---|------------------------------------------|

> |0  |Support for external variables is disabled|

Reviewer: ```suggestion
|`0`  |Support for external variables is disabled|
```

> |1  |Support for external variables is enabled |

Reviewer: ```suggestion
|`1`  |Support for external variables is enabled |
```

> The default value of the parameter is 1.

Reviewer 2: Default is 0

> The default value of the parameter is 0.

Reviewer: ```suggestion
The default value of the parameter is `0`.
```

!!! Info "Information"

>     Support for external variables has been deprecated and is scheduled for removal in a future release. External variables are not supported by default; setting this parameter to 1 reinstates support for the time being.

Reviewer: ```suggestion
    Support for external variables has been deprecated and is scheduled for removal in a future release. External variables are not supported by default; setting this parameter to `1` reinstates support for external variables until it is removed.
```
